tests/paths.rs

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/.
+
+extern crate hglib;
+
+use crate::hglib::{hg, paths};
+use std::fs::OpenOptions;
+use std::io::Write;
+
+mod common;
+
+#[test]
+fn test_basic() {
+    let mut c = common::TestClient::new("paths_basic", &[]);
+
+    let path = c.get_path(".hg/hgrc");
+
+    let mut file = OpenOptions::new().append(true).open(path).unwrap();
+    assert!(file.write_all(b"[paths]\nfoo = bar\n").is_ok());
+    assert!(file.sync_all().is_ok());
+    c.reopen();
+
+    let paths = hg!(c.client, paths).unwrap();
+    let path_map = match paths {
+        paths::Paths::Map(m) => m,
+        _ => panic!("paths return {:?} expected paths::Paths::Map type", paths),
+    };
+
+    assert_eq!(path_map.len(), 1);
```

Comment:
the test is failing locally because of my `~/.hgrc` which contains a paths section with 2 entries (and 1 != 3).
We could use this workaround: https://bz.mercurial-scm.org/show_bug.cgi?id=3147#c2.
So you could add `env::set_var("HGRCPATH", "");` in https://github.com/mozilla/hglib-rust/blob/master/tests/common/mod.rs#L28

Reply:
add `env::set_var("HGRCPATH", "");` into `mod.rs`